csrc/kernel_cache.h

```diff
@@ -247,6 +253,12 @@ class TORCH_CUDA_CU_API FusionKernelRuntime {
   //! Executors holding compiled kernels
   std::vector<FusionExecutor> executors_;
 
+  std::vector<std::unique_ptr<Fusion>> fusions_;
```

Comment:
Why is a `vector` of fusions being added  to the `FusionKernelRuntime`?  A comment would be good here.